indexing-service/src/main/java/org/apache/druid/indexing/overlord/sampler/SamplerConfig.java

```diff
@@ -22,34 +22,53 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.Preconditions;
+import org.apache.druid.data.input.InputFormat;
+import org.apache.druid.data.input.InputSource;
+import org.apache.druid.segment.indexing.DataSchema;
+
+import javax.annotation.Nullable;
 
 public class SamplerConfig
 {
   private static final int DEFAULT_NUM_ROWS = 200;
   private static final int MAX_NUM_ROWS = 5000;
   private static final int DEFAULT_TIMEOUT_MS = 10000;
 
+
+
   private final int numRows;
   private final int timeoutMs;
 
+  private final long maxBytesInMemory;
+
+  private final long maxClientResponseBytes;
+
   @JsonCreator
   public SamplerConfig(
-      @JsonProperty("numRows") Integer numRows,
-      @JsonProperty("timeoutMs") Integer timeoutMs
+      @JsonProperty("numRows") @Nullable Integer numRows,
+      @JsonProperty("timeoutMs") @Nullable Integer timeoutMs,
+      @JsonProperty("maxBytesInMemory") @Nullable Long maxBytesInMemory,
+      @JsonProperty("maxClientResponseBytes") @Nullable Long maxClientResponseBytes
```

Comment:
```suggestion
      @JsonProperty("maxClientResponseBytes") @Nullable HumanReadableBytes maxClientResponseBytes
```

Reply:
yeah, that seems reasonable, forgot about that thing 👍